scripts/utils/getQualityUserScores.ts

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios'
```

Comment:
there's all ready a dumping ground file for celo pg stuff https://github.com/divvi-xyz/divvi-protocol/blob/main/src/celoPGRewards.ts